tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from validators import ValidationFailure
+
+from fds.utils import get_git_repo_name_from_url, construct_dvc_url_from_git_url_dagshub, is_url
+
+
+class TestFds(unittest.TestCase):
+
+    def test_get_git_repo_name_from_url(self):
+        repo_name = get_git_repo_name_from_url("git@github.com:DAGsHub/fds.git")
+        assert repo_name == "fds"
+
+    def test_get_git_repo_name_from_url_from_https(self):
+        repo_name = get_git_repo_name_from_url("https://github.com/DAGsHub/fds.git")
+        assert repo_name == "fds"
```

Comment:
Should also check the valid URL of `https://github.com/DAGsHub/fds`

Reply:
I'll remove this part, i'll get handle this

Reply:
I'll remove this part, i'll let GIT handle this